packages/mui-material/src/locale/index.ts

```diff
@@ -2486,6 +2486,231 @@ export const mkMK: Localization = {
+      },
+    },
+    MuiTablePagination: {
+      defaultProps: {
+        getItemAriaLabel: (type) => {
+          if (type === 'first') {
+            return 'Pergi ke halaman pertama';
+          }
+          if (type === 'last') {
+            return 'Pergi ke halaman terakhir';
+          }
+          if (type === 'next') {
+            return 'Pergi ke halaman seterusnya';
+          }
+          // if (type === 'previous') {
+          return 'Pergi ke halaman sebelumnya';
+        },
+        labelRowsPerPage: 'Baris setiap halaman:',
+        labelDisplayedRows: ({ from, to, count }) =>
+          `${from}–${to} daripada ${count !== -1 ? count : `lebih daripada ${to}`}`,
+      },
+    },
+    MuiRating: {
+      defaultProps: {
+        getLabelText: (value) => {
+          const lastDigit = value % 10;
+          return `${value} Bintang${lastDigit === 1 ? 's' : ''}`;
+        },
+        emptyLabelText: 'kosong',
+      },
+    },
+    MuiAutocomplete: {
+      defaultProps: {
+        clearText: 'Jelas',
+        closeText: 'tutup',
+        loadingText: 'Memuatkan…',
+        noOptionsText: 'Tiada pilihan',
+        openText: 'Buka',
+      },
+    },
+    MuiAlert: {
+      defaultProps: {
+        closeText: 'tutup',
+      },
+    },
+    MuiPagination: {
+      defaultProps: {
+        'aria-label': 'Navigasi penomboran',
+        getItemAriaLabel: (type, page, selected) => {
+          if (type === 'page') {
+            return `${selected ? '' : 'Pergi ke '}muka surat ${page}`;
+          }
+          if (type === 'first') {
+            return 'Pergi ke halaman pertama';
+          }
+          if (type === 'last') {
+            return 'Pergi ke halaman terakhir';
+          }
+          if (type === 'next') {
+            return 'Pergi ke halaman seterusnya';
+          }
+          // if (type === 'previous') {
+          return 'Pergi ke halaman sebelumnya';
+        },
+      },
+    },
+  },
+};
+
+// Nepali-नेपाली
+export const neNP: Localization = {
+  components: {
+    MuiBreadcrumbs: {
+      defaultProps: {
+        expandText: 'बाटो देखाउनुहोस्',
+      },
+    },
+    MuiTablePagination: {
+      defaultProps: {
+        getItemAriaLabel: (type) => {
+          if (type === 'first') {
+            return 'पहिलो पृष्ठमा जानुहोस्';
+          }
+          if (type === 'last') {
+            return 'अन्तिम पृष्ठमा जानुहोस्';
+          }
+          if (type === 'next') {
+            return 'अर्को पृष्ठमा जानुहोस्';
+          }
+          // if (type === 'previous') {
+          return 'अघिल्लो पृष्ठमा जानुहोस्';
+        },
+        labelRowsPerPage: 'प्रति पृष्ठ पङ्क्तिहरू:',
+        labelDisplayedRows: ({ from, to, count }) =>
+          `${from}–${to} को ${count !== -1 ? count : `धेरै ${to}`}`,
+      },
+    },
+    MuiRating: {
+      defaultProps: {
+        getLabelText: (value) => {
+          const lastDigit = value % 10;
+          return `${value} तारा${lastDigit === 1 ? 's' : ''}`;
```

Comment:
Could you double-check if this is correct? I have no clue about Nepali language but it's weird to see an "s" among letters of its native alphabet.

Reply:
I think I made a mistake, the correct letter should be like this स